README.md

```diff
@@ -28,10 +64,15 @@ The Docker instance of Core exposes multiple ports for different types of commun
 | 8087           | Websocket used by the HMI to communicate with SDL Core               |
 | 12345          | SDL Core's port used to communicate with mobile application over TCP |
 
+
 ## Start the HMI
 
+Once SDL Core is setup, follow these steps to clone, build, and run the SDL Generic HMI.
+
 Clone this repository
 
+Note: If you are not making any changes to the generic hmi, you may skip straight to the last step and launch the Generic HMi in a web browser.
```

Comment:
Should update the first reference from "generic hmi" to "Generic HMI" and the second reference from "HMi" to "HMI"